infra-cdk/lib/stacks/IAMStack.ts

```diff
@@ -20,19 +22,24 @@ export class IAMStack extends cdk.Stack {
         this.iamPolicy = new iam.Policy(this, `CL311Policy-${props.stage}-${props.tenant}`, {
             policyName: `CL311Policy-${props.stage}-${props.tenant}`,
             statements: [
-                // Allow access to DynamoDB table
+                // Allow access to DynamoDB table and S3 Buckets
```

Comment:
Can we have one policy, but two separate policy statements, for DynamoDB and S3 buckets? It's confusing to have a policy statement give dynamodb permissions to S3 buckets :)

It would look like this:

```
[
    new iam.PolicyStatement({
        actions: [
            'dynamodb:Scan',
            'dynamodb:GetItem',
            'dynamodb:PutItem',
            'dynamodb:Query',
        ],
        resources: [
            `arn:aws:dynamodb:${ENVIRONMENT.region}:${ENVIRONMENT.account}:table/${props.tableName}`,
            `arn:aws:dynamodb:${ENVIRONMENT.region}:${ENVIRONMENT.account}:table/${props.tableName}/index/*`,
            `arn:aws:dynamodb:${ENVIRONMENT.region}:${ENVIRONMENT.account}:table/${props.reportsTableName}`,
            `arn:aws:dynamodb:${ENVIRONMENT.region}:${ENVIRONMENT.account}:table/${props.reportsTableName}/index/*`,
        ],
    }),
    new iam.PolicyStatement({
        actions: [
            's3:GetObject',
            's3:PutObject',
            's3:DeleteObject',
        ],
        resources: [
            `arn:aws:s3:::${props.bucketStack.bucket.bucketName}`,
            `arn:aws:s3:::${props.bucketStack.bucket.bucketName}/*`,
        ],
    }),
]
```